src/domain/boxMesh.cpp

```diff
@@ -61,14 +61,14 @@ REGISTER(ablate::domain::Domain, ablate::domain::BoxMesh,
          "\n### 3D:\n\n"
          "| Direction | Description | Value |\n"
          "| --- | --- |-----|\n"
-         " | y- | bottom | 1 |\n"
-         " | y+ | top    | 2|\n"
-         " | z+ | front  | 3|\n"
-         " | z- | back   | 4|\n"
+         " | y- | bottom | 3 |\n"
```

Comment:
These were based upon https://gitlab.com/petsc/petsc/-/blob/main/src/dm/impls/plex/plexcreate.c#L867 so our definition of front/back/top might be different.  Would you mind looking at the code to check?  Also you have front/back as both 1

Reply:
I see now. So in the code "front and back " are y+ and y-. So, I should change the definition of the directions equivalently.